repo/packages/S/spark/12/marathon.json.mustache

```diff
@@ -29,8 +29,29 @@
         "HISTORY_CLEANER_INTERVAL": "{{history-server.cleaner-interval}}",
         "HISTORY_CLEANER_MAXAGE": "{{history-server.cleaner-maxage}}",
         "HISTORY_KRB_PRINCIPAL": "{{history-server.kerberos.principal}}",
-        "HISTORY_KRB_KEYTAB": "{{history-server.kerberos.keytab}}"
+        "HISTORY_KRB_KEYTAB": "{{history-server.kerberos.keytab}}",
+{{#security.mesos.authentication.secret_name}}
+        "DCOS_SERVICE_ACCOUNT_CREDENTIAL": { "secret": "serviceCredential" },
+        "MESOS_MODULES": "file:///opt/mesosphere/etc/mesos-scheduler-modules/dcos_authenticatee_module.json",
+        "MESOS_AUTHENTICATEE": "com_mesosphere_dcos_ClassicRPCAuthenticatee",
+{{/security.mesos.authentication.secret_name}}
+{{#security.mesos.ssl.enabled}}
+        "SSL_CERT_FILE": "/run/dcos/pki/tls/certs/scheduler.crt",
+        "SSL_KEY_FILE": "/run/dcos/pki/tls/private/scheduler.key",
+        "SSL_CA_DIR": "/run/dcos/pki/CA/certs",
```

Comment:
Need to make sure these are world-readable (or 'nobody'-readable) when spark is not launched as root.